notesapi/v1/tests/test_views.py

```diff
@@ -1,31 +1,37 @@
+# -*- coding: utf-8 -*-
 import jwt
+import unittest
 from calendar import timegm
 from datetime import datetime, timedelta
 from mock import patch
-import unittest
 
+from django.core.management import call_command
 from django.core.urlresolvers import reverse
 from django.conf import settings
+from django.http import QueryDict
 
 from rest_framework import status
 from rest_framework.test import APITestCase
 
-from annotator import es, auth
-from annotator.annotation import Annotation
 from .helpers import get_id_token
+from notesapi.v1.models import Note
+
+TEST_USER = "test_user_id"
+
+if not settings.ES_DISABLED:
+    import haystack
```

Comment:
can we add something to the README about when an operator would choose to run with this setting?  again, it's good that this flexibility exists, but I've always assumed it would be needed under exceptional circumstances (ES having problems) vs. as a deliberate configuration.